test/fixtures/for/in-auto-const/actual.js

```diff
@@ -1 +1 @@
-for k in {}: k
+for const k in {}: k
```

Comment:
Similarly, since there's a different implementation/codepath for many of the bits related to variables in for loops (even in mainstream Babylon), would probably be good to have tests specific to `for-in` and `for-of` asserting that an error is thrown if a variable isn't prefixed